modules/cli/src/main/scala/scala/cli/launcher/LauncherOptions.scala

```diff
@@ -0,0 +1,21 @@
+package scala.cli.launcher
+
+import caseapp._
+import upickle.default.{ReadWriter, macroRW}
+
+@HelpMessage("Set the ScalaCLI version")
+final case class LauncherOptions(
+  @Group("Launcher")
+  scalaCliVersion: Option[String] = None,
+  @Group("Launcher")
+  @HelpMessage("The version of Scala on which scalaCLI was published")
+  @ValueDescription("2.12|2.13|3")
+  publishedScalaVersion: Option[String] = None
```

Comment:
Maybe we could use something shorter here too?
```suggestion
  cliScalaVersion: Option[String] = None
```